src/loader/utils/load_in_batches.php

```diff
@@ -29,6 +29,13 @@ function load_in_batches(array $config, array $events, callable $loader) {
 
     // Flags events that weren't transformed successfully as events that didn't load.
     $failedtransformevents = filter_transformed_events($events, false);
+
+    // Add error code.
+    foreach ($failedtransformevents as $failed) {
+        $failed["event"]->errortype = get_string('failed_transform_errortype', 'logstore_xapi');
+        $failed["event"]->response =  get_string('failed_transform_response', 'logstore_xapi');
```

Comment:
1. remove extra space before get_string.
2. perhaps default values should be defined in this script and shouldn't be lang string.

Reply:
I have removed the extra space before get_string.